templates/css-variables.twig

```diff
@@ -3,6 +3,12 @@ Later we will use these variables in our source scss files (with a fallback to t
  to expose variables to theme configuration files, thereby removing the need of separate selectors.
  #}
 <style>
+	:root {
+		{% if css_vars['header-primary-font'] %}
+			--campaigns-header-primary-font: {{ css_vars['header-primary-font'] }};
```

Comment:
Just noticed this after approving, but probably better to not prefix this variable (and all other variables) with "campaign".

Reply:
Indeed! Will change it

Reply:
Changed here and in the master-theme.